lib/collection/variable-scope.js

```diff
@@ -98,6 +98,20 @@ _.assign(VariableScope.prototype, /** @lends VariableScope.prototype */ {
     },
 
     /**
+     * Using this function, one can sync the values of this variable list from an existing VariableScope JSON.
+     *
+     * @param {Object} obj A JSON representation of a VariableScope instance, obtained via a .toJSON call.
+     * @param {Boolean=} [track] A flag that indicates whether or not to track synced variables.
+     *
+     * @returns {Object}
+     */
+    syncVariablesFromScope: function (obj, track) {
+        return this.values.syncFromObject(_.transform(obj.values, function (agg, record) {
+            agg[record.key] = { value: record.value, type: record.type };
```

Comment:
disabled variables?

also, since the function says sync from scope, we should check that if it is already a scope instance, we handle that too.